src/WindowPostMessageStream.ts

```diff
@@ -7,8 +7,8 @@ import {
 interface WindowPostMessageStreamArgs {
   name: string;
   target: string;
-  targetWindow?: Window;
-  targetOrigin?: string;
+  targetWindow: Window;
+  targetOrigin: string;
```

Comment:
Sometimes, I give terrible advice:
```suggestion
  targetWindow?: Window;
  targetOrigin?: string;
```

Reply:
The defaults don't mean a thing when you're passing arguments to the constructor 🙃